languages/kotlin/sdk/build.gradle

```diff
@@ -73,19 +73,38 @@ publishing {
     }
     repositories {
         maven {
-            name = "GitHubPackages"
-            url = "https://maven.pkg.github.com/bitwarden/sdk"
+            name = 'GitHubPackages'
+            url = 'https://maven.pkg.github.com/bitwarden/sdk'
             credentials {
-                username = System.getenv("GITHUB_ACTOR")
-                password = System.getenv("GITHUB_TOKEN")
+                username = System.getenv('GITHUB_ACTOR')
+                password = System.getenv('GITHUB_TOKEN')
             }
         }
     }
 }
 
+// Find and include the classes.jar from platform-verifier.
+//
+// Based on the instructions from the readme in https://github.com/rustls/rustls-platform-verifier
+// and issue details from https://github.com/rustls/rustls-platform-verifier/issues/115.
+File findRustlsPlatformVerifierClassesJar() {
+    def dependencyText = providers.exec {
+        it.workingDir = new File('../../')
+        commandLine('cargo', 'metadata', '--format-version', '1', '--manifest-path', 'crates/bitwarden-uniffi/Cargo.toml')
+    }.standardOutput.asText.get()
+
+    def dependencyJson = new groovy.json.JsonSlurper().parseText(dependencyText)
+    def manifestPath = file(dependencyJson.packages.find { it.name == "rustls-platform-verifier-android" }.manifest_path)
+
+    def aar = fileTree(manifestPath.parentFile).matching {
+        include "maven/rustls/rustls-platform-verifier/*/rustls-platform-verifier-*.aar"
+    }.getSingleFile()
+    return zipTree(aar).matching { include 'classes.jar'}.getSingleFile()
+}
+
```

Comment:
I'm definitely not an expert of gradle build, but this seems decent. It's also nice to know that Signal does something similar, and that it's working for them.

What will happen if this fails? Will the build fail, or will the build continue without the rustls-platform-verifier classes bundled?

Reply:
If we can't find classes.jar it's going to fail.